big_tests/tests/graphql_domain_SUITE.erl

```diff
@@ -0,0 +1,304 @@
+    Result1 = execute_auth(#{query => delete_domain_call(), variables => Vars,
+                   operationName => <<"M1">>}, Config),
+    ParsedResult1 = ok_result(<<"domains">>, <<"removeDomain">>, Result1),
+    ?assertEqual(#{<<"msg">> => <<"Domain removed!">>,
+                   <<"domain">> => #{<<"domain">> => <<"exampleDomain">>}},
+                   ParsedResult1),
+
+    Vars2 = #{domain => <<"exampleDomain2">>, hostType => ?HOST_TYPE},
+    Result2 = execute_auth(#{query => delete_domain_call(), variables => Vars2,
+                   operationName => <<"M1">>}, Config),
+    ParsedResult2 = ok_result(<<"domains">>, <<"removeDomain">>, Result2),
+    ?assertEqual(#{<<"msg">> => <<"Domain removed!">>,
+                   <<"domain">> => #{<<"domain">> => <<"exampleDomain2">>}},
+                   ParsedResult2).
+
+get_domains_after_deletion(Config) ->
+    Vars = #{hostType => ?HOST_TYPE},
+    Result = execute_auth(#{query => get_domains_by_host_type_call(),
+                   variables => Vars,
+                   operationName => <<"Q1">>}, Config),
+    ParsedResult = ok_result(<<"domains">>, <<"domainsByHostType">>, Result),
+    ?assertEqual([], ParsedResult).
+
+create_domain_call() ->
+    <<"mutation M1($domain: String!, $hostType: String!)
+           {domains
+               {addDomain(domain: $domain, hostType: $hostType)
+                   {
+                       domain
+                       hostType
+                       enabled
+                   }
+               }
+           }">>.
+
+enable_domain_call() ->
+    <<"mutation M1($domain: String!)
+           {domains
+               {enableDomain(domain: $domain)
+                   {
+                       enabled
+                       domain
+                   }
+               }
+           }">>.
+
+disable_domain_call() ->
+    <<"mutation M1($domain: String!)
+           {domains
+               {disableDomain(domain: $domain)
+                   {
+                       enabled
+                       domain
+                   }
+               }
+           }">>.
+
+get_domains_by_host_type_call() ->
+    <<"query Q1($hostType: String!)
+           {domains
+               {domainsByHostType(hostType: $hostType)}
+           }">>.
+
+get_domain_details_call() ->
+    <<"query Q1($domain: String!)
+           {domains
+               {domainDetails(domain: $domain)
+                   {
+                       domain
+                       hostType
+                       enabled
+                   }
+               }
+           }">>.
+
+delete_domain_call() ->
+    <<"mutation M1($domain: String!, $hostType: String!)
+           {domains
+               {removeDomain(domain: $domain, hostType: $hostType)
+                   {
+                       msg
+                       domain
+                           {domain}
+                   }
+               }
+           }">>.
+
+%% Helpers
+
+execute_auth(Body, Config) ->
```

Comment:
You can now import `execute_auth` from `graphql_helper`, then `get_listener_opts` can be removed.